components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1865,22 +1869,11 @@ void RewardsServiceImpl::GetPublisherInfo(
       return DeferCallback(FROM_HERE, std::move(callback),
```

Comment:
I think we want to remove the condition above this line, so that if the ledger process is not connected, we'll always just do the `return DeferCallback` thing.

Reply:
Fixed in e63215cfbb11475a2d950311715154ff937ca143